MCTSampleApp/MCTSampleApp/AppDelegate.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AppDelegate.swift
+//  MCTSampleApp
+//
+//  Copyright © 2019 Sage Bionetworks. All rights reserved.
```

Comment:
Are these old files that just got moved/copied from elsewhere? or do the copyrights need updating?

Reply:
Yeah, they were accidentally deleted and are re-added.